net.rocketpool/image/package.json

```diff
@@ -1,25 +1,30 @@
 {
   "name": "rocketpool-fetch",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "type": "module",
   "description": "",
-  "main": "dist/app.js",
+  "main": "dist/src/app.js",
   "scripts": {
-    "start": "tsc && node dist/app.js",
+    "start": "tsc && node --es-module-specifier-resolution=node .",
     "build": "tsc",
+    "watch": "tsc -w",
+    "generate-types": "typechain --target=ethers-v5 'abi/*.json'",
     "lint": "eslint . --ext .ts",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "Kamu Data Inc.",
   "license": "Apache-2.0",
   "dependencies": {
-    "abi-decoder": "^2.4.0",
-    "web3": "^1.7.3"
+    "envalid": "8.0.0",
+    "ethers": "5.7.2"
```

Comment:
Can you explain the motivation behind switching to `ethers`?

Reply:
@sergiimk ,
I have worked with both and my conclusions are as follows:
- ethers contains better logic for re-requesting in case of network problems
- ethers has a more flexible interface and type-safe interface
- ethers is generally used as a more up-to-date replacement for web3js

Since the migration in this case took 10 minutes max, I decided to do it

Reply:
Back then I also saw more positive sentiment towards `ethers`, but that specific API that allows filtering events by contract address is what made me go with `web3js` in the end.

Based on all what we discussed I'm glad we made the switch, but in future it's better to communicate such changes ahead of time to confirm what was the reason to pick one or another library. We don't want the next developer try to migrate this back to `web3js` if they have more experience with it :)

Reply:
> Based on all what we discussed I'm glad we made the switch, but in future it's better to communicate such changes ahead of time to confirm what was the reason to pick one or another library.

@sergiimk  yes, agree -- really could have been more transparent on my part -- that's what I'll do next time